lib/install/management/network.go

```diff
@@ -95,26 +99,44 @@ func (d *Dispatcher) removeNetwork(conf *config.VirtualContainerHostConfigSpec)
 		return nil
 	}
 
-	name := conf.Name
-	if network, err := d.session.Finder.Network(d.ctx, name); err != nil || network == nil {
-		log.Infof("Didn't find network %q", name)
-		log.Debugf("Didn't find network for %s", err)
-		return nil
+	br := conf.ExecutorConfig.Networks["bridge"]
+	if br == nil {
+		return fmt.Errorf("Bridge Network ID unknown")
+	}
+	name := br.Network.ID
+	log.Debugf("Remove bridge network based on %s", name)
+
+	moref := new(types.ManagedObjectReference)
+	ok := moref.FromString(name)
+	if !ok {
+		return fmt.Errorf("Unable to delete port group - failed to get moref from: %q", name)
+	}
+
+	net, err := d.session.Finder.ObjectReference(d.ctx, *moref)
+	if err != nil {
+		return fmt.Errorf("Unable to delete port group - failed to find network from: %q", name)
+	}
+	log.Debugf("Delete bridge network: %s", net)
+
+	netw, ok := net.(*object.Network)
+	if !ok {
```

Comment:
if the asserted type is not matching, isn't that really bad? I suspect that error message should be a little more alerting